wedeploy/data/api-p2stats.json

```diff
@@ -0,0 +1,20 @@
+[
+	{
+		"data": true,
+		"description": "Count of all Liferay IDE installs",
+		"method": "GET",
+		"path": "/p2stats"
+	},
+	{
+		"data": true,
+		"description": "Increment install count",
+		"method": "POST",
+		"parameters": {
+			"installCount": {
+				"type": "string",
```

Comment:
Hey @mdelapenya  when we try to put anything other than "type": "string" in this field, like "type": "integer" or "type": "int" we get an error.

Should this be "type": "number"? instead?

Reply:
Hey @gamerson, I think so: https://wedeploy.com/docs/deploy/configuring-deployments/#2